src/chapter-03/template.yaml

```diff
@@ -18,18 +18,18 @@ Resources:
       Architectures:
         - arm64
       Events:
-        Api:
+        CreateShortUrl:
           Type: HttpApi
           Properties:
             Path: /
             Method: POST
-        GetEndpoint:
+        VisitShortUrl:
           Type: HttpApi
           Properties:
-            Path: /{shortUrl}
+            Path: /{linkId}
```

Comment:
Here I thought that `linkId` is a bit more appropriate since we are not receiving the entire short URL but just the ID.